.swiftformat

```diff
@@ -93,6 +93,7 @@
     isEmpty, \ # Rare false positives in situations where a type provides a `count` but not a `isEmpty` property.
     markTypes, \
     noExplicitOwnership, \
+    organizeDeclarations, \
```

Comment:
I find this rule useful, we often see discrepancies in codebase regarding the way we write our // MARK.